beauty-consulting-app/app/Pages/StylistDetailsComplete.js

```diff
@@ -79,26 +21,27 @@ function StylistDetailsComplete() {
             phoneNumber: userContext.phoneNumber,
             stylistDetails: userContext.stylistDetails
         };
-        console.log(req.body);
+
         try {
             const apiUrl = process.env.EXPO_PUBLIC_API_URL;
             if (!apiUrl) {
                 console.error("API URL not defined");
                 return;
             }
-            const res = await axios.post(apiUrl + ':5050/stylist/', req);
```

Comment:
I think this is a good change, but we should put it in a separate PR. Can you remove this change in the code for now i.e. append ":5050" to EXPO_PUBLIC_API_URL in your .env file?

Reply:
Sounds good, I will amend those changes.